frontend/src/features/projects/common/forms/DocumentationForm.tsx

```diff
@@ -17,6 +25,14 @@ const DocumentationForm = ({ isReadOnly, tasks, showNote = false }: IDocumentati
     <Fragment>
       <h3>Documentation</h3>
       <TasksForm tasks={tasks ?? []} isReadOnly={isReadOnly} />
+      {!isReadOnly && (
+        <EmailText>
+          Please send documents to{' '}
+          <a href="mailto:RealPropertyDivision.Disposals@gov.bc.ca">
```

Comment:
so we want this to be a clickable mailto but to just display it with the regular text color?

Reply:
No, it appears blue
![image](https://user-images.githubusercontent.com/33818575/109357498-24bf8e00-7837-11eb-91e2-aea2ae82d5db.png)